src/app/dashboard/revalidation-notes/revalidation-notes.component.ts

```diff
@@ -0,0 +1,39 @@
+import { Component, OnInit } from "@angular/core";
+import { MatBottomSheetRef } from "@angular/material/bottom-sheet";
+
+@Component({
+  selector: "app-revalidation-notes",
+  templateUrl: "./revalidation-notes.component.html",
+  styleUrls: ["./revalidation-notes.component.scss"]
+})
+export class RevalidationNotesComponent implements OnInit {
+  messages: string[]; // mock
+  addEditNote: boolean;
+  constructor(
+    private bottomSheetRef: MatBottomSheetRef<RevalidationNotesComponent>
+  ) {}
+
+  ngOnInit(): void {
+    this.messages = [
```

Comment:
maybe one for a separate pr due to time and effort etc but this mock data should make its way to the state via ngxs

Reply:
yup agree